src/App/Controls/CipherViewCell/CipherViewCell.xaml.cs

```diff
@@ -21,8 +21,10 @@ public CipherViewCell()
         {
             InitializeComponent();
 
-            var deviceActionService = ServiceContainer.Resolve<IDeviceActionService>("deviceActionService");
-            _iconColumn.Width = new GridLength(40 * deviceActionService.GetSystemFontSizeScale(), GridUnitType.Absolute);
+            var fontScale = ServiceContainer.Resolve<IDeviceActionService>("deviceActionService").GetSystemFontSizeScale();
+            _iconColumn.Width = new GridLength(40 * fontScale, GridUnitType.Absolute);
+            _iconImage.WidthRequest = 22 * fontScale;
+            _iconImage.HeightRequest = 22 * fontScale;
```

Comment:
Does this `22` need to be the same value as the original `WidthRequest` and `HeightRequest`? If so, we should consider putting it in a `const` or grabbing the value from `_iconColumn` before using it.

Reply:
I'll put it in a cost, we can't grab it from _iconColumn (unless we do the math proportion: 40/22=1.82 => _iconImage.WidthRequest = _iconColumn.Width.PropetyForTheActualValue / 1,82 but we'd need a const here as well and it's simpler with the 22 const) 😄 . I'll make the change